<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XIAControl</class>
 <widget class="QMainWindow" name="XIAControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>368</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>428</width>
    <height>368</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>428</width>
    <height>10000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>XIAControl</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Number of modules:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="numModSB">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>24</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="modTable"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QCheckBox" name="offlineCB">
            <property name="text">
             <string>Offline mode</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="InitButton">
            <property name="text">
             <string>Initialize</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="Exit_button">
          <property name="text">
           <string>Exit modules</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="syncModCB">
          <property name="text">
           <string>Synchronize clocks</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="simStCB">
          <property name="text">
           <string>Start/stop run simultanious</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Boot_button">
          <property name="text">
           <string>Boot modules</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="BLine_button">
          <property name="text">
           <string>Adjust baseline</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="BCut_button">
          <property name="text">
           <string>Adjust baseline cut-off</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Ready_button">
          <property name="text">
           <string>Ready</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>428</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_run_settings"/>
    <addaction name="actionSave_experiment_settings"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionChange_set_file"/>
    <addaction name="actionChange_firmware_file"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <action name="actionOpen_run_settings">
   <property name="text">
    <string>Open experiment settings</string>
   </property>
  </action>
  <action name="actionSave_experiment_settings">
   <property name="text">
    <string>Save experiment settings</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionChange_set_file">
   <property name="text">
    <string>Change set file</string>
   </property>
  </action>
  <action name="actionChange_firmware_file">
   <property name="text">
    <string>Change firmware file</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>XIAControl</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
